upgrade_tests/test_scenarios/test_subscription.py

```diff
@@ -0,0 +1,106 @@
+"""Test for subscription manifest related Upgrade Scenario's
```

Comment:
Remove manifest word here as it can have other things related to subscription entity